src/workspaces/utils.test.ts

```diff
@@ -251,6 +255,56 @@ describe('hasRegionConstraint', () => {
   });
 });
 
+describe('hasPhiTrackingPolicy', () => {
+  it('Returns true if PHI tracking policy exists', () => {
+    const workspace: WorkspaceWrapper = {
+      ...defaultAzureWorkspace,
+      policies: [phiTrackingPolicy],
+    };
+    expect(hasPhiTrackingPolicy(workspace)).toBe(true);
+  });
+
+  it('Returns false if PHI tracking policy does not exist', () => {
```

Comment:
Nit: a good case for a parameterized test?

Reply:
I considered it but decided to just leave the test as-is.